src/software/thunderscope/common/toast_msg_helper.py

```diff
@@ -0,0 +1,59 @@
+from PyQt6.QtWidgets import QWidget
+from pyqttoast import Toast, ToastPreset
+
+
+def show_toast(
+    parent: QWidget, title: str, text: str, timeout: int, preset: ToastPreset
```

Comment:
```suggestion
    parent: QWidget, title: str, text: str, timeout_ms: int, preset: ToastPreset
```
suffixing the `_ms` makes the code self-documenting

Reply:
+ for all the other functions in this file

Reply:
Done